bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-edit.vue

```diff
@@ -261,9 +261,9 @@ export default {
         }
 
         if (!this.currentModule ||
-            this.currentModule.type.indexOf('script') !== 0 ||
-            (this.currentModule.type === 'jsr223.ScriptedAction' &&
-             this.currentModule.configuration.type === 'application/x-ruby')) {
+            !(this.currentModule.type.indexOf('script') === 0 ||
+             (this.currentModule.type === 'jsr223.ScriptedAction' &&
+              this.currentModule.configuration.type === 'application/x-ruby'))) {
```

Comment:
Since the module types to look for are in the end:
1. `script.ScriptAction`
2. `script.ScriptCondition`
3. `jsr223.ScriptedAction`

I suppose we can simplify those by trying to find `.Script`:

`if (!this.currentModule || this.currentModule.type.indexOf('.Script') < 0)`
(or with a regex of matching this in an array of hardcoded known types)

It makes little sense to scope `jsr223.ScriptedAction` to just Ruby as this module type can be used with other languages too.

(Also please mind the DCO check)

Reply:
I just removed this check completely. seems like if a module has a script in its configuration, we should just try and show it. no need for additional checks on the module type or script type

Reply:
Yes, agreed, no other module type has an unrelated `script` configuration parameter and it would make potential (1st or 3rd party) new module types that accept such a parameter to work instantly.